src/api/api.js

```diff
@@ -1,22 +1,35 @@
 import store from '../store/store';
 import axios from 'axios';
 
-const baseURL = 'http://198.199.100.209:5005/api';
-const getHeaders = () => {
+// const baseURL = 'http://198.199.100.209:5005/api';
+const baseURL = 'http://localhost:5005/api/v1.0';
+const getConfig = () => {
    return {
-      'headers': {
+      headers: {
          'Content-Type': 'application/json',
+         'Authorization': 'token 10cbb8be976d8db1efcca8c60fd7e3594f74772fd877d36afc825ed6df3b7829'
          // Need to implement login register
          // 'Authorization': store.getState().auth.token
-      }
+      },
+      data: {}
    }
 };
+
 const urls = {
-   charts: baseURL + '/charts'
+   charts: baseURL + '/charts',
+   transactions: baseURL + '/transactions'
 };
 
-const getStockHistory = async (ticker, length) => {
-   return await axios(urls.charts + '/${ticker}?length=${length}');
-}
+export const getStockHistory = async (ticker, length) => {
+   return await axios.get(urls.charts + `/${ticker}?length=${length}`, getConfig());
```

Comment:
Might want urls.charts to also be interpolated in the string so it's `${urls.chart}/${ticker}?length= ... `.

Reply:
Nice!